ledger/acctupdates.go

```diff
@@ -68,6 +68,15 @@ const baseResourcesPendingAccountsBufferSize = 100000
 // is being flushed into the main base resources cache.
 const baseResourcesPendingAccountsWarnThreshold = 85000
 
+// baseBoxesPendingAccountsBufferSize defines the size of the base boxes pending buffer size.
```

Comment:
@algoidurovic and reviewers:  Do we agree the reference to _accounts_ can be removed?

I realize the convention was copied from the _resources_ constant.  I don't see how either set of constants is _account_ related.

Reply:
I agree with you but kept it just in case I missed something. Personally, I have no problem removing the comment.

Reply:
@algoidurovic Sorry - I'm suggesting to rename related variables without referencing _accounts_, rather than rewording the comment.

Reply:
After gaining for familiarity with the code, I believe the reference to accounts can be removed in the variable name. Boxes are totally decoupled from accounts at this level, unlike resources, so the convention doesn't need to carry over.

Reply:
@algoidurovic Resolving - Thanks for the rename in https://github.com/algorand/go-algorand/pull/4275/commits/f40283e66555baa0835b5aca70b1782c87493c3a.